creation/interfaces-and-networks.md

```diff
@@ -733,3 +733,103 @@ spec:
 
 > **Note:** for some NICs (e.g. Mellanox), the kernel module needs to be
 > installed in the guest VM.
+
+### macvtap
+
+In `macvtap` mode, virtual machines are directly exposed to an host networking
+interface, and are connected directly to the L2 network connecting the hosts.
+The device is then passed through into the guest operating system as a char
+device.
+
+#### How to expose host interface to the macvtap device plugin
```

Comment:
You are getting into dirty details of device plugins before showing users how simple this is.

I'd start with Hot to expose host interface to virtual machines instead

Reply:
This is a user guide; I think a user comes here and executes whatever we write sequentially. If you're selecting which interfaces get exposed, this is the order to follow.

I don't think the objective of the user guide is to "sell" an alternative to the user (if I wanted to do that, I'd avoid mentioning the whole device plugin thing - since everything is exposed by default, which simplifies the user's life  by quite a lot).